store/index.js

```diff
@@ -9,13 +9,13 @@ function mapPages(pages) {
   pages.map((page) => {
     switch(page.slug){
       case 'new-york-medical-services':
```

Comment:
Why does it still have this hard-coded?

Reply:
This because it is store based and all data is fetched at the very start to improve performance and that is why it is very fast.
So here we are mapping the pages in separate objects and that particular object is passed when that page is loaded. 
And because we are not fetching the page at page load. That's why had to be mapped into objects initially. 
Does that make sense?

Reply:
Sure, but it needs to be dynamic, not hard-coded.